Userland/Games/Snake/SnakeGame.cpp

```diff
@@ -1,10 +1,10 @@
 /*
- * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
- * Copyright (c) 2021, Mustafa Quraish <mustafa@serenityos.org>
- * Copyright (c) 2022, the SerenityOS developers.
- *
- * SPDX-License-Identifier: BSD-2-Clause
- */
+* Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
```

Comment:
Looks like the indentation of both the files got messed up, maybe your editor is configured for the wrong indentation size?

Reply:
Thanks, CTRL+C, CTRL+V messed it up... :^)